kiam.tf

```diff
@@ -0,0 +1,94 @@
+
+# ##########
+# # THANOS #
```

Comment:
Does this stuff relate to Thanos? If so, it should be removed so that it's added as part of the Thanos change. If not, then this header comment should be removed.

Reply:
Yes, sorry. Missed it, I'll move it out now.